src/components/divs/InfoCard.tsx

```diff
@@ -8,7 +8,7 @@ type Props = {
 export const InfoCard = ({ headline, children }: Props) => {
   return (
     <article className="bg-white rounded-lg shadow-md p-6 max-w-2xl mx-auto">
-      <h2 className=" pb-2 mb-4 text-xl font-semibold">{headline}</h2>
+      <h2 className="uppercase pb-2 mb-4 text-4xl font-semibold font-numbers">{headline}</h2>
```

Comment:
also watchout for the mobile adjustments, since text-4x could look too big on phones (use md: or lg: media queries)